backend/api/pipeline.proto

```diff
@@ -133,6 +143,7 @@ service PipelineService {
 }
 
 message Url {
+  // Online storage URL of the file containing the pipeline or pipeline version definition.
```

Comment:
Suggested revision:

URL of the pipeline definition or the pipeline version definition.

Reply:
done